### PR TITLE
Add integration tests using the retail_mini schema

### PR DESCRIPTION
**Pull request by kaklakariada**
This adds an integration test using the retail_mini schema (https://github.com/exasol/database-migration/blob/master/test/testing_files/retail_mini/retail_mini.sql).

### CODE REVIEW

File: doc/changes/changes_1.5.6.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Row Level Security lua 1.5.6, released 2024-??-??
+
+Code name:
+
+## Summary
+
+## Features
+
+* ISSUE_NUMBER: description
```
Comment (redcatbear) [inline doc/changes/changes_1.5.6.md:9]: If you don't have a ticket, you can use the PR number here.

File: src/test/java/com/exasol/RetailSchemaIT.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            assertRlsQueryWithUser("SELECT count(*) FROM " + virtualSchema.getName() + "." + table, user,
+                    table().row(0L).matches());
+        }
+    }
+
+    private ExasolSchema createRetailSchema() throws IOException, SQLException {
+        final ExasolSchema schema = factory.createSchema("RETAIL_MINI");
+        try (Statement statement = connection.createStatement()) {
+            statement.execute("OPEN SCHEMA " + schema.getName());
+            executeSqlScript(Path.of("src/test/resources/retail_mini.sql"), statement);
```
Comment (redcatbear) [inline src/test/java/com/exasol/RetailSchemaIT.java:40]: The TDDB has this feature built in.

**Review by redcatbear — CHANGES_REQUESTED** [review]

File: doc/changes/changes_1.5.6.md (hunk 1 above)
Comment (kaklakariada) [reply doc/changes/changes_1.5.6.md:9]: That won't work, PK only accepts issues ;)

File: src/test/java/com/exasol/RetailSchemaIT.java (hunk 2 above)
Comment (kaklakariada) [reply src/test/java/com/exasol/RetailSchemaIT.java:40]: Method `executeSqlFile()` from TDDB expects a single SQL statement in the file, but we have multiple statements separated with `;`.
I created https://github.com/exasol/test-db-builder-java/issues/135 to add this feature.

File: doc/changes/changes_1.5.6.md (hunk 1 above)
Comment (ckunki) [reply doc/changes/changes_1.5.6.md:9]: other PR did write "n/a" here

**Pull request merged**